pycloak/middleware.py

```diff
@@ -199,14 +199,14 @@ def store_claim_on_user(self, request, user):
 
             # check for related fields on the user model
             rel_fields = attr["field"].split(".")
-            field = rel_fields.pop(0)
             obj = user
-            for rel_field in rel_fields:
-                obj = getattr(obj, rel_field)
+            field = None
+            for field in rel_fields:
+                obj = getattr(obj, field)
             obj_to_save.add(obj)
 
             # check for value
-            if value := request.jwt_data.get(claim) is None:
+            if (value := request.jwt_data.get(claim)) is None:
```

Comment:
I'd rather remove the `is None` as it is generally not needed, and remove the parentheses, too.

Reply:
We would introduce the case that claim is `""`. Is that relevant?

Reply:
Who am I to decide that? - is a claim with empty value worth to be taken into account? I'd say yes, since I could catch up the empty value later in a callback and do something with it.

Reply:
It's the other way round, sorry. With the test for `is None` we currently allow an empty string as a value. If we change it to the following:
```
value = request.jwt_data.get(claim)
if not value:
    [...]
```
we would raise the exception when claim is an empty string.

So I guess we leave the check for `is None` then? Maybe it's more readable in the following way?
```
value = request.jwt_data.get(claim)
if value is None:
    [...]
```

Reply:
yes, much better

Reply:
Done